### PR TITLE
Adds single and bulk standardisation of full names and acronyms

### PR DESCRIPTION
**Pull request by MaximumEndurance**

### CODE REVIEW

**Review by aCampello — APPROVED** [review]

**Comment by aCampello** [comment]
```bash
--- validators
------- UniversityNameValidator
------- CurrencyValidator
------- etc
--- suggestions
------ fuzzy_matcher
------ ml_matcher
```

**Comment by MaximumEndurance** [comment]
- [x] Run flake8 on code
- [x] Check if pd.read_csv() has caching feature. Otherwise find a way to store the data locally for successive iterations

**Comment by MaximumEndurance** [comment]
Reformatted using flake8.

File: app.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+app = Flask(__name__)
+
+
+@app.route("/")
+def upload():
+    return render_template("home.html")
+
+
+@app.route("/uploadSuccess", methods=["POST"])
+def uploadSuccess():
```
Comment (aCampello) [inline app.py:13]: I think it is ok for the endpoint name, but the function name is usually underscored according to PEP8, e.g. `upload_success`

File: initialise_standard_univ_list.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    if not os.path.exists('GridUnivsData/acronyms.csv'):
+        acronyms = pd.read_csv(f"{path}{sub_path}acronyms.csv")
+        print("Fetch from remote Acronym")
+
+    else:
+        acronyms = pd.read_csv('GridUnivsData/acronyms.csv')
+
+    acronyms = acronyms.rename(columns={"grid_id": "ID"})
+
+    return pd.merge(all_univs, acronyms, on='ID', how='outer')
```
Comment (aCampello) [inline initialise_standard_univ_list.py:28]: Nice fix

File: templates/success.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+{% for table in tables %}
+            {{ table|safe }}
+{% endfor %}
+</body>
+</html>
```
Comment (aCampello) [inline templates/success.html:12]: missing endline

**Review by aCampello — APPROVED** [review]
:shipit: Looks good to me. Minor comments - no need to address those, just wanted to record them

**Pull request merged**